server/libs/domain/src/storage-template/storage-template.service.ts

```diff
@@ -77,9 +77,22 @@ export class StorageTemplateService {
 
       try {
         await this.storageRepository.moveFile(asset.originalPath, destination);
+
+        let sidecarDestination;
+        if (asset.sidecarPath != null) {
```

Comment:
```suggestion
        if (!asset.sidecarPath) {
```

Reply:
This actually creates a typescript error since asset.sidecarPath is string or null, the `!` appears to also include empty string.

![image](https://github.com/immich-app/immich/assets/3135216/b20b889b-42b2-49f7-bd4c-f19a1e8fa1f5)

Reply:
Sorry, I meant to suggest `if(asset.sidecarPath)` (negated)

Reply:
@jrasm91  I'm not sure what you mean here by negating it.

Reply:
Nevermind, I was thinking about it backwards. This is resolved.